packages/auth/src/accessToken/routes.test.ts

```diff
@@ -285,4 +283,67 @@ describe('Access Token Routes', (): void => {
       expect(ctx.response.status).toBe(204)
     })
   })
+
+  describe('Rotation', (): void => {
+    let grant: Grant
+    let token: AccessToken
+    let managementId: string
+
+    beforeEach(
+      async (): Promise<void> => {
+        grant = await Grant.query(trx).insertAndFetch({
+          ...BASE_GRANT
+        })
+        await Access.query(trx).insertAndFetch({
+          grantId: grant.id,
+          ...BASE_ACCESS
+        })
+        token = await AccessToken.query(trx).insertAndFetch({
+          grantId: grant.id,
+          ...BASE_TOKEN
+        })
+        managementId = BASE_TOKEN.managementId
+      }
+    )
+
+    test('Cannot rotate nonexistent token', async (): Promise<void> => {
+      managementId = v4()
+      const ctx = createContext(
+        {
+          headers: { Accept: 'application/json' }
+        },
+        { managementId }
+      )
+
+      await expect(accessTokenRoutes.rotate(ctx)).rejects.toMatchObject({
+        status: 404,
+        message: 'token not found'
+      })
+    })
+
+    test('Can rotate token', async (): Promise<void> => {
+      const ctx = createContext(
+        {
+          headers: { Accept: 'application/json' }
+        },
+        { managementId }
+      )
+
+      await accessTokenRoutes.rotate(ctx)
+      expect(ctx.response.status).toBe(200)
```

Comment:
Can you also test for the body, the conent-type, and that it is satisfies the API spec (` expect(ctx.response).toSatisfyApiSpec()`)?